lib/media/catalog/index.ts

```diff
@@ -60,63 +61,68 @@ export function isCatalog(catalog: any): catalog is Catalog {
 }
 
 export interface Track {
-	kind: string
-	container: string
+	name: string
 }
 
 export interface Mp4Track extends Track {
-	container: "mp4"
 	init_track: string
 	data_track: string
 }
 
-export interface AudioTrack extends Track {
-	kind: "audio"
-	codec: string
-	channel_count: number
-	sample_rate: number
-	sample_size: number
+export interface SelectionParamsVideo {
+	codec: string;
+	height: number;
+	width: number;
+	frame_rate: number
 	bit_rate?: number
 }
 
+export interface SelectionParamsAudio {
+	bit_rate: number;
+	channel_count: number;
+	codec: string;
+	sample_rate: number;
+	sample_size: number;
+}
+
+
+export interface AudioTrack extends Track {
+	name: "audio"
+	selectionParams: SelectionParamsAudio;
+}
+
 export interface VideoTrack extends Track {
-	kind: "video"
-	codec: string
-	width: number
-	height: number
-	frame_rate: number
-	bit_rate?: number
+	name: "video"
```

Comment:
This is the track name, not the kind. We can't use `"audio"` or `"video"` because that would only allow one track of each type.

There's no equivalent to `kind` in draft-02, which I think is a big oversight. There is only supposed to be a single `SelectionParams` type instead of splitting them into audio/video.